### ISSUE TITLE
Suppress 'endpoint is not synced' when 'Couldn't connect to any eth1 node'

### ISSUE THREAD

**Issue by SeaMonkey82** [sigp/lighthouse#2488]
## Description

```
WARN Eth1 endpoint is not synced
CRIT Couldn't connect to any eth1 node.
```

Those two messages together is kind of confusing to me.  One implies that the endpoint isn't synced and the other implies that it isn't reachable at all.  It seems like the WARN message should be suppressed if the CRIT message is true, as the beacon node can't know the state of the endpoint if it is unable to connect to it.

**Comment by pawanjay176** [comment]
@SeaMonkey82 The crit has been changed to say
`Could not connect to a suitable eth1 node` in https://github.com/sigp/lighthouse/pull/2461

I think the new error message addresses your concern. Let us know if it can be improved even further.

**Comment by paulhauner** [comment]
Thanks @SeaMonkey82 and @pawanjay176. I'll close this, assuming it's resolved in #2461. Feel free to re-open if you disagree :)

**Comment by come-maiz** [comment]
When following a guide to test the merge :point_up: I still got confused about this.

I would like to reopen it. My geth node is syncing, so lighthouse should be patient, instead of showing me a critical error that takes me to wonder if the eth1 client is wrong somehow.
